### ISSUE TITLE
docs: Issue on page Examples

### ISSUE THREAD

**Issue by pogzie** [scylladb/scylla-manager#3879]
I would like to report an issue on page https://manager.docs.scylladb.com/branch-3.2/repair/examples

### Problem

Example on https://manager.docs.scylladb.com/stable/repair/examples.html#schedule-a-weekly-repair using the Docker example doesnt work. Also, if basing from the Docker example, it might be best to standardize the use of the cluster name. 

```
scylla-manager@354a2ccb890d:/$ sctool tasks -c test
+------------------------+--------------+--------+----------+---------+-------+------------------------+------------+--------+------------------------+
| Task                   | Schedule     | Window | Timezone | Success | Error | Last Success           | Last Error | Status | Next                   |
+------------------------+--------------+--------+----------+---------+-------+------------------------+------------+--------+------------------------+
| healthcheck/alternator | @every 15s   |        | UTC      | 32      | 0     | 10 Jun 24 04:35:09 UTC |            | DONE   | 10 Jun 24 04:35:24 UTC |
| healthcheck/cql        | @every 15s   |        | UTC      | 32      | 0     | 10 Jun 24 04:35:09 UTC |            | DONE   | 10 Jun 24 04:35:24 UTC |
| healthcheck/rest       | @every 1m0s  |        | UTC      | 8       | 0     | 10 Jun 24 04:35:09 UTC |            | DONE   | 10 Jun 24 04:36:09 UTC |
| repair/all-weekly      | 0 23 * * SAT |        | UTC      | 0       | 0     |                        |            | NEW    | 15 Jun 24 23:00:00 UTC |
+------------------------+--------------+--------+----------+---------+-------+------------------------+------------+--------+------------------------+
scylla-manager@354a2ccb890d:/$ sctool repair -c test -s now+4h -i 7d
Flag --start-date has been deprecated, use cron instead
Flag --interval has been deprecated, use cron instead
Error: create repair target: create repair plan: no replicas to repair
Trace ID: LlgQjyfjTnW4hiTQuluHOg (grep in scylla-manager logs)
```

###  Suggest a fix

If we're pushing the reader to not use deprecated flags, this should also be fixed by using `--cron`. Either way, using cron and following the example it errors out. Further investigation required. 

```
scylla-manager@354a2ccb890d:/$ sctool repair -c test --cron @daily
Error: create repair target: create repair plan: no replicas to repair
Trace ID: vUKn8bgGTn621SWtw-CVGQ (grep in scylla-manager logs)
```

**Comment by Michal-Leszczynski** [comment]
@pogzie sorry for such a late response!
I will remove deprecated flags from the example section. I will also rename the cluster name in docker example.
In terms of the error, do you remember used SM version? It works fine for me with SM 3.3, Scylla 6.0 locally with docker setup.

**Comment by Michal-Leszczynski** [comment]
This issue was fixed as a part of #3917.
